manifest/x86_64/t/telegram.filelist

```diff
@@ -1,2 +1,2 @@
-/usr/local/bin/telegram
 /usr/local/bin/Telegram
```

Comment:
This shouldnt be happening... we should have consistent sorting orders now. Is there anything you are doing to change the way filelists are created?

Reply:
No, it's just sorting the way YOU changed in https://github.com/chromebrew/chromebrew/pull/9429. :)